fem/bilininteg_hcurl.cpp

```diff
@@ -1691,6 +1691,7 @@ static void SmemPACurlCurlApply3D(const int D1D,
                                   const Vector &x,
                                   Vector &y)
 {
+#ifdef MFEM_DEVICE_COMPILE
```

Comment:
It turns out we botched this: the above `#ifdef` is always false in this function because it is a host function.

I guess we should moving the `#ifdef` inside the lambda.

Reply:
I'll do some compilation tests to understand better what is happening.

Reply:
I'm confused, if it is a CPU issue, why the CI did not catch it?

Reply:
I think we don't run PA tests on CPU in CI.